ui/src/utils/api.js

```diff
@@ -0,0 +1,353 @@
+export default createApi;
```

Comment:
Might want to consider breaking up these API calls into separate files so the API calls are organized better..
 
/utils/api/
 - index.js
 - reference.js
 - content.js
 - branch.js 
 
etc ...

Reply:
Hey Tim, thanks for the review. This file is auto-generated by a script. Hence it being all in one messy file. The process hasn't been automated yet but will be in a follow up PR